atomics/T1069.001/T1069.001.yaml

```diff
@@ -37,4 +37,38 @@ atomic_tests:
     command: |
       get-localgroup
       Get-LocalGroupMember -Name "Administrators"
-    name: powershell
\ No newline at end of file
+    name: powershell
+- name: SharpHound3 - LocalAdmin
+  auto_generated_guid: e03ada14-0980-4107-aff1-7783b2b59bb1
+  description: |
+    This module runs the Windows executable of SharpHound in order to remotely list members of the local Administrators group (SAMR)
+  supported_platforms:
+  - windows
+  input_arguments:
+    domain_name:
+      description: FQDN of the targeted domain
+      type: string
+      default: DOMAIN.CORP
+    sharphound_path:
+      description: SharpHound Windows executable
+      type: path
+      default: '$env:TEMP\SharpHound.exe'
+  dependency_executor_name: powershell
+  dependencies:
+  - description: |
+      SharpHound binary must exist on disk and at specified location (#{sharphound_path}).
+      And the computer must be domain joined (implicit authentication).
+    prereq_command: |
+      $sharphound_path = cmd /c echo #{sharphound_path}
+      if (Test-Path $sharphound_path) { exit 0 } else { exit 1 }
+    get_prereq_command: |
+      $sharphound_path = cmd /c echo #{sharphound_path}
+      Invoke-WebRequest "https://github.com/BloodHoundAD/BloodHound/blob/e062fe73d73c015dccb37fae5089342d009b84b8/Collectors/SharpHound.exe?raw=true" -OutFile $sharphound_path
+  executor:
+    name: powershell
+    elevation_required: false
+    command: |
+      & "#{sharphound_path}" -d  #{domain_name} --CollectionMethod LocalAdmin --NoSaveCache
+      Write-Host "End of SharpHound attack"
+    cleanup_command: |
+      Remove-Item #{sharphound_path}
```

Comment:
Cleanup shouldn't remove the stuff install by get_prereq, but instead it should delete temp files generated by the execution.
Here for example, it would be the cache file (but not needed since `--NoSaveCache`) and the output file. You can use the `--OutputDirectory` option to store everything in a well known temp folder that you can cleanup here